applications/rubintv/values-tucson-teststand.yaml

```diff
@@ -0,0 +1,3 @@
+fontend:
+  config:
+    debug: true
```

Comment:
Is this in use? Some things below make me think it's not, so it should probably be deleted.

Reply:
No, TTS isn't there and has been deleted.